gameLogic/GameMap.java

```diff
@@ -9,133 +9,108 @@
 
 public class GameMap implements Drawable {
 	private int currentLevel;
-	private HashMap<Integer, ImageIcon[][]> levelMapping = new HashMap<>();
-	private HashMap<Integer, Rectangle[]> hitRectangleMapping = new HashMap<>();
-
-	public GameMap() {
-		this(1);
-	}
+	private int currentMap;
+	private HashMap<String, ImageIcon[][]> levelMapping = new HashMap<>();
+	private HashMap<String, Rectangle[]> hitRectangleMapping = new HashMap<>();
+
+	public static final int TOTAL_LEVELS = 1;
+	public static final int MAPS_PER_LEVEL = 4;
+	private static final int MAP_WIDTH = 10;
```

Comment:
Let's rename MAP_WIDTH and MAP_HEIGHT to something that includes "tiles" so MAP_WIDTH_TILES etc. I think it would be confusing from the outside when you try to access a final int called MAP_WIDTH and expect 1000 instead of 10